CMakeLists.txt

```diff
@@ -1,90 +0,0 @@
-# Copyright 2019-2020 CERN and copyright holders of ALICE O2.
```

Comment:
what is this file and why was it removed?

Reply:
Sorry, I delete this CMakeList.txt by mistake. I will upload it.